desc/compute/_profiles.py

```diff
@@ -1474,3 +1474,33 @@ def _current_r(params, transforms, profiles, data, **kwargs):
 def _current_rr(params, transforms, profiles, data, **kwargs):
     data["current_rr"] = 2 * jnp.pi / mu_0 * data["I_rr"]
     return data
+
+
+@register_compute_fun(
+    name="shear",
+    label="\\rho \\partial_{\\rho} \\iota / \\iota",
+    units="~",
+    units_long="None",
+    description="Normalized derivative of the rotational transform",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    data=["0", "iota", "iota_r", "iota_rr", "rho"],
+    axis_limit_data=["iota_den_r", "iota_num_r"],
+)
+def _shear(params, transforms, profiles, data, **kwargs):
+    eps = 1e2 * jnp.finfo(jnp.array([1.0]).dtype).eps
```

Comment:
Is there a better way to do this?

Reply:
might be more understandable if you use `data['iota'].dtype`

Reply:
Done. But I meant about having this `eps` defined inside the compute function like this.

Reply:
yeah thats pretty standard. The threshold depends on the application and the data, so a global one usually doesn't work.